glyphs-reader/src/font.rs

```diff
@@ -1439,7 +1470,7 @@ impl RawFont {
 
     /// `<See https://github.com/schriftgestalt/GlyphsSDK/blob/Glyphs3/GlyphsFileFormat/GlyphsFileFormatv3.md#differences-between-version-2>`
     fn v2_to_v3(&mut self) -> Result<(), Error> {
-        self.v2_to_v3_weight()?;
+        self.v2_to_v3_master_names()?;
```

Comment:
I'm surprised v2_to_v3 weight was removed as opposed to just a new call being added, just want to confirm that was on purpose?

Reply:
yeah I did on purpose.. the old v2_to_v3_weight wasn't actually setting any `weight` attribute (which is only found in v2, not in v3), it was actually setting the master.name, but only taking into account `weight`. Whereas my code is also taking into account the additional `width` and `custom` attributes of v2 sources (we weren't parsing those before)